storybook/preview-head-vrt.html

```diff
@@ -1,3 +1,17 @@
+<script>
+	// Flag the document as in VRT mode.
+	document.documentElement.dataset.vrt = true;
```

Comment:
I added this as part of a closer integration with Storybook that allows us to do some "if VRT" stuff in there if we want but it was ultimately not needed. This doesn't hurt to keep though and still has some utility as well.